core/src/test/java/org/elasticsearch/index/engine/InternalEngineTests.java

```diff
@@ -1813,11 +1809,9 @@ public void testUpgradeOldIndex() throws IOException {
             assertTrue("[" + indexFile + "] missing index dir: " + src.toString(), Files.exists(src));
             assertTrue("[" + indexFile + "] missing translog dir: " + translog.toString(), Files.exists(translog));
             Path[] tlogFiles = filterExtraFSFiles(FileSystemUtils.files(translog));
-            assertEquals(Arrays.toString(tlogFiles), tlogFiles.length, 1);
+            assertEquals(Arrays.toString(tlogFiles), tlogFiles.length, 2); // ckp & tlog
```

Comment:
side note - we should probable extend our old indices to have more then one translog file sometimes (ongoing recoveries on 2.0 will do this )